forch/file_handlers.py

```diff
@@ -0,0 +1,25 @@
+"""File changes watcher"""
+
+import hashlib
+
+from watchdog.events import FileSystemEventHandler, LoggingEventHandler, FileSystemEvent
+
+import forch.faucetizer
+
+
+class FaucetConfigFileHandler(FileSystemEventHandler):
+    """Handles config file changes"""
+    def __init__(self, faucetizer:forch.faucetizer.Faucetizer, structural_config_file, behavior_config_file):
+        self._faucetizer = faucetizer
+        self._structural_config_file = structural_config_file
+        self._behavior_config_file = behavior_config_file
+
+    def on_modified(self, event):
+        """on file modified"""
+        super(FaucetConfigFileHandler, self).on_modified(event)
+
+        if event.is_directory or self._structural_config_file != event.src_path:
+            return
+
+        forch.faucetizer.update_structural_config(self._faucetizer, self._structural_config_file)
```

Comment:
this is duplicated code/logic with the current periodic mechanism. I think it's good to keep the old functionality for now (remove in another pass if necessary), just in case this new stuff doesn't work in production, but this logic (what happens when something changes) should be captured in one place somewhere.

Reply:
Ok, now it is still periodically writing out behavioral config, while updating structural config is triggered by file change.

Reply:
I don't understand what your point is. It's not a question of when the thing is triggered, it's about what it does when it's triggered...?